content/docs/2.11/scalers/external-push.md

```diff
@@ -15,13 +15,19 @@ triggers:
 - type: external-push
   metadata:
     scalerAddress: external-scaler-service:8080
-    tlsCertFile: /path/to/tls/cert.pem # optional
```

Comment:
We should keep this setting here, just mark it as deprecated and link the issue here: https://github.com/kedacore/keda/issues/4549

Reply:
Sure. Let me make this change.